sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/Models/WhatIfChange.cs

```diff
@@ -36,18 +36,21 @@ public WhatIfChange()
         /// <param name="resourceId">Resource ID</param>
         /// <param name="changeType">Type of change that will be made to the
         /// resource when the deployment is executed. Possible values include:
-        /// 'Create', 'Delete', 'Ignore', 'Deploy', 'NoChange',
-        /// 'Modify'</param>
+        /// 'Create', 'Delete', 'Ignore', 'Deploy', 'NoChange', 'Modify',
+        /// 'Unsupported'</param>
+        /// <param name="unsupportedReason">The explanation about why the
+        /// resource is unsupported by What-If.</param>
         /// <param name="before">The snapshot of the resource before the
         /// deployment is executed.</param>
         /// <param name="after">The predicted snapshot of the resource after
         /// the deployment is executed.</param>
         /// <param name="delta">The predicted changes to resource
         /// properties.</param>
-        public WhatIfChange(string resourceId, ChangeType changeType, object before = default(object), object after = default(object), IList<WhatIfPropertyChange> delta = default(IList<WhatIfPropertyChange>))
+        public WhatIfChange(string resourceId, ChangeType changeType, string unsupportedReason = default(string), object before = default(object), object after = default(object), IList<WhatIfPropertyChange> delta = default(IList<WhatIfPropertyChange>))
```

Comment:
breaking change.  Please add a customization to mitigate.

Reply:
Fixed.